packages/apps/plugins/plugin-deck/project.json

```diff
@@ -1,27 +1,19 @@
 {
   "$schema": "../../../../node_modules/nx/schemas/project-schema.json",
   "name": "plugin-deck",
-  "tags": [
-    "scope:plugins"
-  ],
+  "tags": ["scope:plugins"],
   "sourceRoot": "packages/apps/plugins/plugin-deck/src",
   "projectType": "library",
   "targets": {
     "build": {},
     "compile": {
       "options": {
-        "entryPoints": [
-          "{projectRoot}/src/index.ts",
-          "{projectRoot}/src/meta.ts"
-        ],
-        "platforms": [
-          "browser"
-        ]
+        "entryPoints": ["{projectRoot}/src/index.ts", "{projectRoot}/src/meta.ts"],
+        "platforms": ["browser"]
       }
     },
-    "lint": {}
+    "lint": {},
+    "test": {}
```

Comment:
if it doesn't work quickly don't bother, but if it's just as easy, try using vitest instead. We aspire to move over to vitest and off of our custom test infra and the less there is to port the easier that will be.

Reply:
see https://github.com/dxos/dxos/pull/7383 for an example or `@dxos/random-access-storage`

Reply:
currently, yes